### PR TITLE
JAPI-73 WUAction backwards compatiblity with 5 series ESPs

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner <michael.gardner@lexisnexis.com>

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@rpastrana Please review

File: wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java [hunk 1]
```diff
@@ -2116,7 +2116,17 @@ public boolean doWorkunitAction(String wuid, ECLWUActions action)
...
+        	// fallback to 1.56 api
+        	org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WUAction fwa = new org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WUAction();
+        	fwa.setActionType(action.getValue());
+        	fwa.setWuids(new String[] { wuid });
+        	org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WUActionResponse fwar = this.fallBackWorkunitsServiceSoapProxy.getWsWorkunitsServiceSoap().WUAction(fwa);
+        	if (fwar == null || fwar.getActionResults() == null || fwar.getActionResults().length == 0
+        			|| fwar.getActionResults()[0].getResult() == null
+        			|| !fwar.getActionResults()[0].getResult().equals("Success"))
+        	{
+        		throw new Exception("Unable to perform " + action.getValue() + " on " + wuid);
```
Comment (rpastrana) [inline wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java]: can we provide the actual reason why this failed?

**Comment by dabayliss** [comment]
@rpastrana we are waiting on this - is there a problem ?

**Comment by rpastrana** [comment]
@Michael-Gardner can you make the change from the code review?

**Comment by Michael-Gardner** [comment]
It fell by the wayside, I'll have it in this morning.

**Comment by rpastrana** [comment]
looks fine.

**Comment by Michael-Gardner** [comment]
@dabayliss This change is now live in 1.0.2.  It can be found on the maven central repository https://search.maven.org/#search%7Cga%7C1%7Corg.hpccsystems.wsclient

**Comment by dabayliss** [comment]
Thank you

**Pull request merged**